src/test/kotlin/com/igorwojda/integer/stepsgenerator/challenge.kt

```diff
@@ -4,7 +4,12 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generateSteps(n: Int): List<String> {
-    TODO("not implemented")
+    val pyramidElementsList = mutableListOf<String>()
+    for (element in 0 until n) {
```

Comment:
Can you think of a way of creating the result list while iterating instead of appending to a mutable list?